### ISSUE TITLE
base_action for sequences not accepted by sdwan_custom_control_topology_policy_definition

### ISSUE THREAD

**Issue by rrahimm** [CiscoDevNet/terraform-provider-sdwan#114]
The sequences of the sdwan_custom_control_topology_policy_definition should allow defining base_action as documented below.
However, we see that the plan file doesn't generate value for this field.

[base_action](https://registry.terraform.io/providers/CiscoDevNet/sdwan/0.2.11/docs/resources/custom_control_topology_policy_definition#base_action) (String) Base action, either accept or reject
Choices: accept, reject

```yaml
 # module.sdwan.sdwan_custom_control_topology_policy_definition.custom_control_topology_policy_definition["CCT_DEFINITION_TEST1"] will be created
  + resource "sdwan_custom_control_topology_policy_definition" "custom_control_topology_policy_definition" {
      + default_action = "accept"
      + description    = "CCT_DEFINITION_TEST1"
      + id             = (known after apply)
      + name           = "CCT_DEFINITION_TEST1"
      + sequences      = [
          + {
              + action_entries = [
                  + {
                      + set_parameters = [
                          + {
                              + community = "100:1000"
                              + type      = "community"
                            },
                          + {
                              + community_additive = true
                              + type               = "communityAdditive"
                            },
                          + {
                              + preference = 48
                              + type       = "preference"
                            },
                          + {
                              + omp_tag = 88
                              + type    = "ompTag"
                            },
                          + {
                              + tloc_action = "strict"
                              + type        = "tlocAction"
                            },
                          + {
                              + tloc_color         = "custom2"
                              + tloc_encapsulation = "ipsec"
                              + tloc_ip            = "1.2.5.9"
                              + type               = "tloc"
                            },
                          + {
                              + tloc_list_id      = (known after apply)
                              + tloc_list_version = (known after apply)
                              + type              = "tlocList"
                            },
                          + {
                              + service_tloc_color         = "custom1"
                              + service_tloc_encapsulation = "ipsec"
                              + service_tloc_ip            = "3.4.5.6"
                              + service_type               = "netsvc1"
                              + service_vpn_id             = 62
                              + type                       = "service"
                            },
                        ]
                      + type           = "set"
                    },
                ]
              + id             = 1
              + ip_type        = "ipv4"
              + match_entries  = [
                  + {
                      + color_list_id      = (known after apply)
                      + color_list_version = (known after apply)
                      + type               = "colorList"
                    },
                  + {
                      + community_list_id      = (known after apply)
                      + community_list_version = (known after apply)
                      + type                   = "community"
                    },
                  + {
                      + omp_tag = 65
                      + type    = "ompTag"
                    },
                  + {
                      + expanded_community_list_id      = (known after apply)
                      + expanded_community_list_version = (known after apply)
                      + type                            = "expandedCommunity"
                    },
                  + {
                      + preference = 45
                      + type       = "preference"
                    },
                  + {
                      + originator = "10.10.20.30"
                      + type       = "originator"
                    },
                  + {
                      + site_list_id      = (known after apply)
                      + site_list_version = (known after apply)
                      + type              = "siteList"
                    },
                  + {
                      + site_id = 6789056
                      + type    = "siteId"
                    },
                  + {
                      + path_type = "direct-path"
                      + type      = "pathType"
                    },
                  + {
                      + tloc_list_id      = (known after apply)
                      + tloc_list_version = (known after apply)
                      + type              = "tlocList"
                    },
                  + {
                      + type             = "vpnList"
                      + vpn_list_id      = (known after apply)
                      + vpn_list_version = (known after apply)
                    },
                  + {
                      + prefix_list_id      = (known after apply)
                      + prefix_list_version = (known after apply)
                      + type                = "prefixList"
                    },
                  + {
                      + tloc_color         = "custom1"
                      + tloc_encapsulation = "ipsec"
                      + tloc_ip            = "10.10.33.67"
                      + type               = "tloc"
                    },
                ]
              + name           = "rule1"
              + type           = "route"
            },
          + {
              + action_entries = [
                  + {
                      + set_parameters = [
                          + {
                              + omp_tag = 89
                              + type    = "ompTag"
                            },
                          + {
                              + preference = 49
                              + type       = "preference"
                            },
                        ]
                      + type           = "set"
                    },
                ]
              + id             = 2
              + ip_type        = "ipv4"
              + match_entries  = [
                  + {
                      + carrier = "carrier1"
                      + type    = "carrier"
                    },
                  + {
                      + color_list_id      = (known after apply)
                      + color_list_version = (known after apply)
                      + type               = "colorList"
                    },
                  + {
                      + domain_id = 567
                      + type      = "domainId"
                    },
                  + {
                      + group_id = 678
                      + type     = "groupId"
                    },
                  + {
                      + omp_tag = 77
                      + type    = "ompTag"
                    },
                  + {
                      + originator = "12.13.14.15"
                      + type       = "originator"
                    },
                  + {
                      + preference = 88
                      + type       = "preference"
                    },
                  + {
                      + site_list_id      = (known after apply)
                      + site_list_version = (known after apply)
                      + type              = "siteList"
                    },
                  + {
                      + site_id = 6789056
                      + type    = "siteId"
                    },
                  + {
                      + tloc_list_id      = (known after apply)
                      + tloc_list_version = (known after apply)
                      + type              = "tlocList"
                    },
                  + {
                      + tloc_color         = "custom1"
                      + tloc_encapsulation = "ipsec"
                      + tloc_ip            = "10.10.33.67"
                      + type               = "tloc"
                    },
                ]
              + name           = "rule2"
              + type           = "tloc"
            },
        ]
      + version        = (known after apply)
    }
```

**Comment by danischm** [comment]
I cannot recreate this issue. When I set the base action, I see it in the plan and it is configured correctly.

**Comment by danischm** [comment]
This should be fixed from 0.2.11 release onwards.

**Comment by rrahimm** [comment]
Hi Dan

i tested this and it worked in the latest release.
